modules/siddhi-core/src/main/java/io/siddhi/core/table/holder/IndexEventHolderForCache.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License is incorrect add the correct year

Reply:
Fixed in https://github.com/siddhi-io/siddhi/pull/1579